billing_viewing_invoices.md

```diff
@@ -1,22 +1,16 @@
 View the Status of Current and Past Invoices
 ============================================
 
-You can now view all of your current and past invoices right in the tracking system. To view your invoices, please follow these steps.
+To view the status of invoices: 
 
--   Click on your user name in the upper right corner, then click on 'Organization Info'.
+Select the dropdown in the top right corner, then select "Organization Info".
```

Comment:
and  select "Organization Info."

[comments & refactoring note using ordered lists instead of headings and quotation marks to delineate fields/controls. For now, nitpicky thing if you touch any of these .md files: end punctuation, period or comma, goes within the close quotation mark. Your call if you want to apply this change.